pkg/driverbuilder/builder/templates/alinux.sh

```diff
@@ -42,19 +39,20 @@ rm -Rf /tmp/kernel
 mkdir -p /tmp/kernel
 mv usr/src/kernels/*/* /tmp/kernel
 
+cd {{ .DriverBuildDir }}
+mkdir -p build && cd build
+{{ .CmakeCmd }}
```

Comment:
Since we always use the same CmakeCmd, make it easily customizable (at least during development) by using a single templated variable for it.